tomb/src/types/config/bucket/omni.rs

```diff
@@ -286,19 +280,20 @@ impl Display for OmniBucket {
         );
 
         // If we have both present
-        if let Some(local) = &self.local && let Some(remote) = &self.remote {
-            info = format!("{info}\nname:\t\t{}\norigin:\t\t{}\nremote_id:\t{}\ntype:\t{}\nstorage_class:\t{}",
+        if let Some(local) = &self.local
+            && let Some(remote) = &self.remote
+        {
+            info = format!(
+                "{info}\nname:\t\t{}\norigin:\t\t{}\nremote_id:\t{}\ntype:\t{}\nstorage_class:\t{}",
                 local.name,
                 local.origin.display(),
                 remote.id,
                 remote.r#type,
                 remote.storage_class
             );
-        }
-        else if let Some(local) = &self.local {
+        } else if let Some(local) = &self.local {
             info = format!("{info}\n{}", local);
-        }
-        else if let Some(remote) = &self.remote {
+        } else if let Some(remote) = &self.remote {
```

Comment:
This is slightly different from before but is much harder to follow with the extra complexity in the else if statements.